dlv crashed when tracing a pointer type of args or returns under --ebpf mode

dlv trace can not capture the pointer type of args or returns under ebpf mode, my testing is below.

Delve Debugger: Version: 1.21.0

go version: go1.18.7 linux/amd64

OS:  Debian 10 GNU/Linux 

Linux Kernel: 5.10.135

Codes like this : 

func logic3(x int64, y *int64) interface{} {
    return (x + x + *y)
}

sudo root firstly, and using command like "dlv trace logic3 --ebpf", you can got output like ：
===================
panic: Internal debugger error: runtime error: invalid memory address or nil pointer dereference
runtime.gopanic (0x4352e6)
 	/usr/local/go/src/runtime/panic.go:838
runtime.panicmem (0x44b1dc)
 	/usr/local/go/src/runtime/panic.go:220
runtime.sigpanic (0x44b1ac)
 	/usr/local/go/src/runtime/signal_unix.go:818
github.com/go-delve/delve/pkg/proc.(*Target).GetBufferedTracepoints.func1 (0x73d642)
 	~/go/pkg/mod/github.com/go-delve/delve@v1.21.0/pkg/proc/target.go:457
github.com/go-delve/delve/pkg/proc.(*Target).GetBufferedTracepoints (0x73d354)
 	~/go/pkg/mod/github.com/go-delve/delve@v1.21.0/pkg/proc/target.go:472
github.com/go-delve/delve/service/debugger.(*Debugger).GetBufferedTracepoints (0x7fd3e4)
 	~/go/pkg/mod/github.com/go-delve/delve@v1.21.0/service/debugger/debugger.go:2238
github.com/go-delve/delve/service/rpc2.(*RPCServer).GetBufferedTracepoints (0x866a64)
 	~/go/pkg/mod/github.com/go-delve/delve@v1.21.0/service/rpc2/server.go:147
reflect.Value.call (0x4c6844)
 	/usr/local/go/src/reflect/value.go:556
reflect.Value.Call (0x4c5dbe)
 	/usr/local/go/src/reflect/value.go:339
github.com/go-delve/delve/service/rpccommon.(*ServerImpl).serveJSONCodec.func2 (0x9506c5)
 	~/go/pkg/mod/github.com/go-delve/delve@v1.21.0/service/rpccommon/server.go:340
github.com/go-delve/delve/service/rpccommon.(*ServerImpl).serveJSONCodec (0x94faac)
 	~/go/pkg/mod/github.com/go-delve/delve@v1.21.0/service/rpccommon/server.go:342
runtime.goexit (0x465720)
 	/usr/local/go/src/runtime/asm_amd64.s:1571
goroutine 45 [running]:
github.com/go-delve/delve/cmd/dlv/cmds.traceCmd.func1.1()
 	~/go/pkg/mod/github.com/go-delve/delve@v1.21.0/cmd/dlv/cmds/commands.go:734 +0x7f0
created by github.com/go-delve/delve/cmd/dlv/cmds.traceCmd.func1
 	~/go/pkg/mod/github.com/go-delve/delve@v1.21.0/cmd/dlv/cmds/commands.go:726 +0x10f2


I expect to see the logic3's arguments and returns, but it's not work. so, please check it or told me how to fix, thx!

--- a/pkg/proc/target.go
+++ b/pkg/proc/target.go
@@ -451,6 +451,11 @@ func (t *Target) GetBufferedTracepoints() []*UProbeTraceResult {
 		v.Addr = ip.Addr
 		v.Kind = ip.Kind
 
+		if v.RealType == nil {
+			v.Unreadable = errors.New("type not supported by ebpf")
+			return v
+		}
+
 		cachedMem := CreateLoadedCachedMemory(ip.Data)
 		compMem, _ := CreateCompositeMemory(cachedMem, t.BinInfo().Arch, op.DwarfRegisters{}, ip.Pieces, ip.RealType.Common().ByteSize)
 		v.mem = compMem

